nengo_viz/static/viz_modal.js

```diff
@@ -22,7 +22,13 @@ VIZ.Modal.footer = function(type){
 
     if (type === "close") {
         $footer.append('<button type="button" class="btn btn-default" data-dismiss="modal">Close</button>');
-    } else {
+    } 
+    else if (type === 'confirm_reset') {
+        $footer.append('<button type="button" id="confirm_reset_button" class="btn btn-primary">Reset</button>');
+        $footer.append('<button type="button" class="btn btn-default" data-dismiss="modal">Close</button>');
+        $('#confirm_reset_button').on('click', function() {toolbar.reset_model_layout();});
```

Comment:
No, it's merged, just do it in another PR. #186 will have to be rebased to master, and it does the same thing, so we can do it there.